cfme/tests/automate/test_file_import.py

```diff
@@ -31,3 +37,55 @@ def test_domain_import_file(import_datastore, import_data):
             3. Import should work. Check imported or not.
     """
     assert import_datastore.exists
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1720611])
+@pytest.mark.parametrize("files", ["datastore_blank.zip", "dialog_blank.yml"],
+                         ids=["datastore", "dialog"])
+@pytest.mark.uncollectif(lambda files: files == "dialog_blank.yml" and
+                         BZ(1720611, forced_streams=['5.10']).blocks)
```

Comment:
I don't see that https://bugzilla.redhat.com/show_bug.cgi?id=1720611 will be fixed, there's no clone to 5.10, so you can always uncollect based on the appliance version.

Reply:
This BZ might get back-ported to 5.10. Hence keeping it as it is.